ledger/testing/consensusRange.go

```diff
@@ -88,14 +99,12 @@ func TestConsensusRange(t *testing.T, start, stop int, test func(t *testing.T, v
 	cfg := config.GetDefaultLocal()
 	for i := start; i <= stop; i++ {
 		version := versionStringFromIndex(i)
+		disable := rand16(t)%2 == 0
 		t.Run(fmt.Sprintf("cv=%s", version), func(t *testing.T) {
```

Comment:
need to log disable value. Easiest way is to add into into the test name as I suggested earlier.

Reply:
my bad, will add to test run log